docs/source/api/regions_name_name.rst

```diff
@@ -0,0 +1,153 @@
+..
```

Comment:
if an endpoint is being deprecated we shouldnt/dont need to add documentation around it

Reply:
Okay, rebased and documentation removed in [`f22b0e9c3d~`...`c954e8776b`](https://github.com/apache/trafficcontrol/compare/f22b0e9c3d~...c954e8776b).

Reply:
Yeah, there's no point in doing that since we don't want people to use them. It's a waste of time, but providing the docs are well-formed and accurate I wouldn't hold up a PR because of adding docs, IMO. Reviewer's discretion, I suppose.